game/template/easy_level1.html

```diff
@@ -1,63 +1,25 @@
 <!DOCTYPE html>
 <html>
 <head>
-<style>
-body {
-  background-color: lightblue;
-}
-h1 {
-  color: darkblue;
-  text-align: center;
-}
-.button-left {
-  background-color: #4CAF50; /* Green */
-  border: none;
-  color: white;
-  position: relative;
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  margin: 4px 2px;
-  cursor: pointer;
-}
-
-.button-right {
-  background-color: #CD5C5C;
-  border: none;
-  color: #8B4513;
-  position: relative;
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  margin: 4px 10px;
-  cursor: pointer;
-}
-/* .button2 {background-color: #008CBA;} Blue */
-/* .button3 {background-color: #f44336;} Red  */
-/* .button4 {background-color: #e7e7e7; color: black;} Gray  */
-/* .button5 {background-color: #555555;} Black */
-h1 {
-  co
-}
-</style>
+  <title>DAVINCI GAME</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="easy.css">
 </head>
 <body>
 
-<!-- <h2>Button Colors</h2>
-<p>Change the background color of a button with the background-color property:</p> -->
-
 <!-- <button class="button">Green</button> -->
-<a href= "https://www.facebook.com/" class="button-left">Linktofacebook</a>
-<a href= "https://www.youtube.com/" class="button-right">Linktoyoutube</a>
+<a href= "https://www.facebook.com/" class="button-left">Main Menu</a>
+<a href= "https://www.youtube.com/" class="button-right">Next</a>
 <h1>DAVINCI GAME</h1>
```

Comment:
I think the header should be change into the game question number and specify the stage such as Easy 1.